src/Core/LanguageService.fs

```diff
@@ -473,7 +512,8 @@ module LanguageService =
         let verbose = "FSharp.verboseLogging" |> Configuration.get false
 
         let spawnNetCore dotnet =
-            let ionidePluginPath = VSCodeExtension.ionidePluginPath () + "/bin_netcore/fsautocomplete.dll"
+            //let ionidePluginPath = VSCodeExtension.ionidePluginPath () + "/bin_netcore/fsautocomplete.dll"
+            let ionidePluginPath = @"C:\proj\ionide-vscode-fsharp\paket-files\github.com\fsharp\FsAutoComplete\src\FsAutoComplete\bin\Debug\netcoreapp2.1\fsautocomplete.dll"
```

Comment:
Without this nothing works. This obviously needs to be reverted. But what is the current development story here with lsp?

Reply:
I have patched `build.fsx` that targets my local clone of FSAC. I do changes in FSAC, run `build.cmd LocalRelase` in FSAC repo, and then run `build.cmd Build` in Ionide repo to copy latest built version of FSAC

Reply:
Ok, I think we should see if we can improve that, actually not having to restart code was a really nice feature of fsacUrl. But I guess I just have to get used to the new model (just some days after I started liking the old one :D)
So its probably more personal frustration than anything else ;)

Reply:
One other solution would be to make this path a configuration option, then we could just configure the Debug path of FSAC similar to fsacUrl. What do you think?

Reply:
> One other solution would be to make this path a configuration option, then we could just configure the Debug path of FSAC similar to fsacUrl. What do you think?

Yes, I think this would be good

Reply:
Let's handle it in other PR

Reply:
Ok Thanks